src/test/resources/smoke-suite.xml

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd">
+<suite name="Smoke suite" thread-count="1">
```

Comment:
you don't need to put a `thread-count` option when you're not scaling your tests